lib/reporter.js

```diff
@@ -161,7 +161,11 @@ var generateReport = function (options) {
         var screenShotDirectory;
         suite.reportAs = 'Features';
 
-        screenShotDirectory = options.output ? path.join(options.output, '..', 'screenshot') : 'screenshot/';
+        if(options.screenshotsDir) {
+            screenShotDirectory = options.output ? path.join(options.output, '..', options.screenshotsDir) : options.screenshotsDir;
+        } else {
+            screenShotDirectory = options.output ? path.join(options.output, '..', "screenshots") : "screenshots";
```

Comment:
This is nice. It is much better to say screenshot**s**. BTW - it's minor, can you update the double quotes to single?